enzoic/enzoic.py

```diff
@@ -537,22 +590,80 @@ def get_user_passwords(self, username: str, include_exposure_details: bool = Fal
         Returns a list of passwords that Enzoic has found for a specific user.  This call must be enabled for your
         account or you will receive a 403 rejection when attempting to call it.
         see https://docs.enzoic.com/enzoic-api-developer-documentation/api-reference/credentials-api/cleartext-credentials-api
-        :param username: The username you wish to receive the a list of passwords for.
+        :param username: The username you wish to receive a list of passwords for.
         :param include_exposure_details: Includes the details of the exposure the password was found in if True,
         :return:
         """
         # username needs to be converted to lowercase and url encoded
-        params = {
+        query_params = {
             "username": str(username).lower(),
             "includePasswords": 1,
         }
+
         if include_exposure_details:
-            params["includeExposureDetails"] = int(include_exposure_details)
+            query_params["includeExposureDetails"] = int(include_exposure_details)
 
-        result = urlencode(params, quote_via=quote_plus)
+        result = urlencode(query_params, quote_via=quote_plus)
 
         response = self._make_rest_call(
-            self.api_base_url + self.ACCOUNTS_API_PATH + f"?{result}", "GET", None)
+            self.api_base_url + f"/cleartext-credentials?{result}", "GET", None
+        )
+        if response.status_code == 404:
+            return False
+        else:
+            return response.json()
+
+    def get_user_passwords_by_partial_hash(self, username: str, include_exposure_details: bool = False) -> Union[bool, Dict]:
+        """
+        Returns a list of passwords that Enzoic has found for a specific user.  This call must be enabled for your
+        account or you will receive a 403 rejection when attempting to call it.
+        see https://docs.enzoic.com/enzoic-api-developer-documentation/api-reference/credentials-api/cleartext-credentials-api
+        :param username: The username you wish to receive a list of passwords for.
+        :param include_exposure_details: Includes the details of the exposure the password was found in if True,
+        :return:
+        """
+        # username needs to be converted to sha256 partial hash and url encoded
+        query_params = {
+            "partialUsernameHash": hashing.calc_sha256_unsalted_hash(str(username).lower())[:8],
+            "includePasswords": 1,
```

Comment:
Not necessary to include the includePasswords param if the /cleartext-credentials-by-partial-hash endpoint is used.